plugins/fcp_can_c/templates/rpc_device_c.j2

```diff
@@ -0,0 +1,35 @@
+#include "{{ device_name_snake }}_rpc.h"
+#include "{{ device_name_snake }}_can.h"
+#include <string.h>
+
+/*---------------------------------------- RPC Dispacher -----------------------------------------*/
+bool can_service_dispatch(const CanFrame *frame, void (*send)(const CanFrame *)) {
+
+    if (frame->dlc != sizeof(RpcMessage)) {
+        return false;
+    }
```

Comment:
Not sufficient to distinguish normal messages from rpc calls.

We need to check rpc_get ids for every possible caller

```c
bool valid_rpc_call = false;
for (int i=0; i<number_of_devices; i++) {
    if (frame->sid == rpc_get_ids[i]) {
         valid_rpc_call = true;
    }
}

if (!valid_rpc_call) {
    return false;
}
```
^
| pseudocode